src/components/ChefOfTheWeekSection/ChefOfTheWeekSection.tsx

```diff
@@ -1,43 +1,84 @@
-import React from "react";
 import { SectionTitle } from "../SectionTitle/SectionTitle";
-import "./ChefOfTheWeekSection.scss";
+import styles from "./ChefOfTheWeekSection.module.scss";
+import ChefsRestaurantCard from "../ChefsRestaurantCard/ChefsRestaurantCard";
+import { Chef } from "../../types/types";
+import data from "../../data/data.json";
+import useGetScreenWidth from "../../hooks/useGetWidthScreen";
+import { UIConstants } from "../../shared/constants";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+
+import goToIcon from "../../assets/images/goToIcon.svg";
 
 export const ChefOfTheWeekSection = () => {
+  const chefs: Chef[] = data.data.chefs;
+  const chefOfTheWeek: Chef | undefined = chefs.find(
+    (chef) => chef.name === "Yossi Shitrit"
+  );
+  const screenWidth: number = useGetScreenWidth();
+  const isMobile: boolean = screenWidth <= UIConstants.sizes.mobileWidth;
+  const isTablet: boolean =
+    UIConstants.sizes.mobileWidth < screenWidth &&
+    screenWidth <= UIConstants.sizes.tabletWidth;
```

Comment:
This is getting repetitive. Don't you think it would be better to extract both `isMobile` and `isTablet` into their own hooks that use the original `useGetScreenWidth`?
Try to figure out the logic I implemented there and recreate and build on top of it.

Reply:
@nitzanpap  done